pkg/dockerregistry/server/pullthroughblobstore.go

```diff
@@ -118,6 +118,10 @@ func (r *pullthroughBlobStore) proxyStat(ctx context.Context, retriever importer
 }
 
 // ServeBlob attempts to serve the requested digest onto w, using a remote proxy store if necessary.
+// Important! This function is called for GET and HEAD requests. Docker client uses[1] HEAD request
```

Comment:
@smarterclayton is this a "bug" in the mirroring? (iow. should we disable mirroring on HEAD request?)